### PR TITLE
Sending tab

### PR DESCRIPTION
**Pull request by pi0neerpat**
# Depends on #9 

## Sending table

image

image

## Added \<ProviderCard />

Displays provider details with a link to site, and etherscan (currently hard-coded to Kovan)

image

## Other

- Added more icons

## `/helpers/utils.js`

- added `getShortAddress(address)` which returns the human-friendly text "0xabc2...d74e3"
- Added `getEtherscanLink(address, network)` which returns the appropriate anchor tag

### CODE REVIEW

File: src/dashboard/Sending/Sending.js [hunk 1]
```diff
@@ -1,11 +1,145 @@
...
+            headings={[
+              'Recipient',
+              'Tribute Amount',
+              'Current APR',
+              'Flowing Since',
+              'Total Sent',
+              'Actions'
+            ]}
+            rows={[
+              [getEtherscanLink('1', 'kovan'), 2, 3, 4, 5, 6],
```
Comment (ejwessel) [inline src/dashboard/Sending/Sending.js]: can we put this in a resource somewhere so when we change it we don't have to go hunt for this?

File: src/dashboard/helpers/general.js [hunk 2]
```diff
@@ -29,3 +29,29 @@ export const CONTRACTS = {
...
+    tags: ['game', 'paid', 'entertainment'],
+    image: 'superMariano'
+  }
+};
+
+export const NETWORKS = {
+  kovan: {
+    etherscan: 'https://kovan.etherscan.io/address/'
+  }
+};
```
Comment (ejwessel) [inline src/dashboard/helpers/general.js]: This file feels like it's misnamed
`helper/general.js` I think it's better to denote these as constants no?

File: src/dashboard/Sending/Sending.js (hunk 1 above)
Comment (pi0neerpat) [reply src/dashboard/Sending/Sending.js]: This is going to get changed once we have context implemented. Just putting it there now so we remember to use it.

File: src/dashboard/helpers/general.js (hunk 2 above)
Comment (pi0neerpat) [reply src/dashboard/helpers/general.js]: done

**Pull request merged**